docs/robots/husky/tutorials_husky.mdx

```diff
@@ -4,7 +4,1535 @@ sidebar_position: 4
+7. On first boot, the username will be `administrator` and the password will be `clearpath`. You should use the
+   `passwd` utility to change the `administrator` account password.
+8. To set up a factory-standard Husky robot, ensure all your peripherals are plugged in, and run the following command:
+
+    ```bash
+    rosrun husky_bringup install
+    ```
+
+    The install script will configure a ros upstart service, that will bring up the base Husky launch files on boot.
+    The script will also detect any standard peripherals (IMU, GPS, etc.) you have installed and add them to the service.
+
+#### Testing Base Configuration
+
+1. To test your configuration, start the background service with the following command:
+
+    ```bash
+    sudo systemctl start ros
+    ```
+
+2. The `COMM` light on your Husky should go from red to green. You can check that the service has started correctly
+   by checking the logs:
+
+    ```bash
+    sudo journalctl -u ros
+    ```
+
+3. Your Husky should now be accepting commands from your joystick. The service will automatically start each time you
+   boot your Husky's PC.
+
+#### Pairing the Controller    {#pairing-the-controller}
+
+##### PS4 Controller
+
+<ComponentPs4ControllerPairing />
+
+##### Logitech F710 Controller
+
+Some Husky robots ship with a Logitech F710 controller instead of a PS4 controller.  Pairing these controllers
+is very easy: simply plug the USB dongle into one of robot's USB ports and turn the controller on.
+
+By default Husky will use the PS4 controller for teleoperation and ignore the F710.  To enable the F710 to control
+the robot, run `sudo nano /etc/ros/setup.bash` and add the following line to the middle of the file, under the six
+`#` characters:
+
+```bash
+######
+export HUSKY_LOGITECH=1
+```
+
+Save the file and quit nano.  Then restart ROS by running `sudo systemctl restart ros` or rebooting the robot.
+When ROS restarts it will now use the Logitech controller as its teleoperation input device.
+
+### Setting up Husky's Network Configuration    {#husky-networking}
+
+Husky is normally equipped with a combination Wifi + Bluetooth module.
+If this is your first unboxing, ensure that Husky's wireless antennas are firmly screwed on to the chassis.
+Some Husky robots may only be equipped with a single antenna, depending on the exact model of PC installed in the robot.
+
+#### First Connection
+
+By default, Husky's wifi is in client mode, looking for the wireless network at the Clearpath factory.
+
+<ComponentWiredRobotConnection />
+
+#### Changing the Default Password
+
+<ComponentChangingDefaultPassword />
+
+#### Wifi Setup
+
+<ComponentWifiRobotConnection />
+
+### Installing Remote PC Software    {#remote-pc-software}
+
+:::note
+This step is optional.
+:::
+
+It is often convenient to use a Remote PC (eg. laptop) to command and observe your robot. To do this, your
```

Comment:
We have called the 'Remote PC' a 'Desktop computer' or 'Development computer' on other pages.
Should we have a standard term?